tests/python_nested_if/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -21,6 +33,17 @@ elif x == 12:
 else:
     print(123)
 
+if foo:
+    foo()
+elif bar:
+    bar()
+elif baz:
```

Comment:
It's not clear to me that this is what we want. Certainly, if I wrote the original code and it got reformatted to this, I would be frustrated. I think it can make sense to have a single `if` in an `else` block, particularly if you're planning to extend it later.

I think to fix this we'd need to look at the `source` of the `If` node, given how the AST is desugared. In particular, I think in the above `is_elif` or `get_elif_node` function I describe, we could add a check along the lines of `n.orelse[0].source.startswith("elif")`.